src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -87,7 +88,8 @@ export default function TripPattern({
   return (
     <div className={style.tripPatternContainer}>
       <motion.div
-        role={'button'}
+        id="assistant-details-accordion"
```

Comment:
This will probably cause screen reader and HTML errors as it should be prefixed by ID? IIRC, this component is included several places, and ID have to be unique. Common practise for this us using the `useId` React hook